File: BasicCourseOOP/1. CursoPOOUber/Java/main.java
class Main {
    public static void main(String[] args) {
        /*Car car = new Car("PQE4651",new Account("Juan Perez", "1754658987"));
        car.passengers=4;
        car.printDataCar();

        Car car2 = new Car("PMA3644",new Account("Pedro Rosero", "1728935463"));
        car2.passengers=3;
        car2.printDataCar();*/

        UberX uberX=new UberX("PLK7858",new Account("Juan Perez", "1754658987"), "Ford", "Mustan");
        //uberX.passengers=4;
        uberX.setPassenger(2);
        uberX.printDataCar();
        
        UberVan uberVan=new UberVan("PLK7858",new User("Diego Paz", "1754548789", "diegopaznaula@gmail.com", "password"));
        uberVan.setPassenger(6);
        uberVan.printDataCar();
    }
}

File: BasicCourseOOP/1. CursoPOOUber/Python/main.py
from car import Car
from account import Account
from uberX import UberX
from uberPool import UberPool
from user import User

if __name__ == "__main__":
    # car=Car("PTD9631", Account("Pedro Rosero","1745876523"))
    # print(vars(car))
    # print(vars(car.driver))
    uberX=UberX("PLL","Pedro Rosero","Toyota","Corona")
    print(vars(uberX))
    uberPool=UberPool("PHJ2332","Juan Perez","Ford","Mustan")
    #print(vars(uberPool.driver))

    user=User("Diego Paz","1745878987","diegopaznaula@gmail.com","password")
    print(vars(user))